app/forms/hyrax/forms/collection_form.rb

```diff
@@ -92,14 +100,24 @@ def member_work_ids
         end
 
         def collection_member_service
-          @collection_member_service ||= membership_service_class.new(scope: self, collection: collection, params: nil)
+          @collection_member_service ||= membership_service_class.new(scope: self, collection: collection, params: blacklight_config.default_solr_params)
```

Comment:
The defaults should be set by a search builder and should not be passed in

Reply:
I'm confused about what you want here. You approved this when I refactored the collection membership searches into the service, so what is different now? 

Additionally, since this is used in four different places (two collections controllers, the admin set controller, and the collection form), I think this should be a separate issue if you want additional refactoring. It is beyond the scope of this specific issue.